### PR TITLE
Add Mastodon & Bluesky handle support to author profiles

### PR DESCRIPTION
**Pull request by zeppelin**
Mastodon example:

image

Bluesky example:

image

### CODE REVIEW

**Comment by zeppelin** [comment]
@marcoow If this looks fine, I'll ask & add everyone else's handle, too to this PR, if they want to

**Comment by marcoow** [comment]
sure, this is good – we could add Bluesky as well

File: src/authors/nickschot.md [hunk 1]
```diff
@@ -2,5 +2,6 @@
 name: "Nick Schot"
 github: nickschot
 twitter: nickschot
+mastodon: "@nickschot@toot.community"
```
Comment (nickschot) [inline src/authors/nickschot.md:6]:
```suggestion
bluesky: nickschot.bsky.social
mastodon: "@nickschot@toot.community"
```

File: src/authors/marcoow.md [hunk 2]
```diff
@@ -3,5 +3,6 @@ name: "Marco Otte-Witte"
 github: marcoow
 twitter: marcoow
 linkedin: marco-otte-witte
+mastodon: "@marcoow@mastodon.social"
```
Comment (marcoow) [inline src/authors/marcoow.md:6]:
```suggestion
mastodon: "@marcoow@mastodon.social"
bluesky: marcoow.bsky.social
```

**Review by marcoow — APPROVED** [review]

**Pull request merged**